models/maisi_ct_generative/scripts/sample.py

```diff
@@ -923,7 +923,7 @@ def ensure_output_size_and_spacing(self, labels, check_contains_target_labels=Tr
                         raise ValueError(
                             (
                                 f"Resampled mask does not contain required class labels {anatomy_label}. "
-                                "Please tune spacing and output size."
+                                "Please consider increasing the spacing or specifying a larger output size."
```

Comment:
"increase the output spacing"

Reply:
Thanks, update this part in the latest commit.